src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -84,118 +85,207 @@ void doNamesMatch_No_Apostrophe_NameCombined() { // Note: In the future, we may
      ******************************************* Compare Street Address Tests ******************************************
      *******************************************************************************************************************/
     @Test
-    void doAddressesMatch_Yes_IgnoreCase() {
-        assertEquals(AnswerType.yes, CompareContactParts.doAddressesMatch("123 main Street", "123 Main street"));
+    void doStreetAddressesMatch_Yes_IgnoreCase() {
+        Contact contactOne = new Contact();
+
+        contactOne.getAddress().setStreetAddress("123 main Street");
+        assertEquals(AnswerType.yes, CompareContactParts.doStreetAddressesMatch(contactOne, "123 Main street"));
```

Comment:
I think you are overloading this test class, aren't you? It is set up to test CompareContactParts (the static methods) but you are now adding tests that are creating/using full Contacts. I was expecting those kinds of tests to be in the ContactTest class. If it isn't clear what I mean, I can show you what I was thinking on Monday. My concern is that your tests/classes are getting more dependent on one another which will make troubleshooting and maintenance more challenging.

Reply:
We'll talk more about this. But I'm going to go ahead and implement the streetAddressMatch(). Worst case scenario, if I have to decouple, I will only have to change the tests.